portfolio/src/main/webapp/script.js

```diff
@@ -75,3 +75,70 @@ function addQuoteToDom(quote) {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+const google = window.google;
+window.onload = createMap;
+let map = window.map;
```

Comment:
```suggestion
let map;
```

Reply:
Fixed! At first I was getting errors when trying to make map a global var and window.map was the only thing that helped but I look it out and it still works fine.

Reply:
That makes sense. You still need the `let map;` in the outer scope so that the variable name is defined in the right place.